python/solid_dmft/dmft_tools/initial_self_energies.py

```diff
@@ -421,12 +428,29 @@ def determine_dc_and_initial_sigma(general_params, advanced_params, sum_k,
                     # if magmom positive the up channel will be favored
                     for spin_channel in sum_k.gf_struct_solver[icrsh].keys():
                         if 'up' in spin_channel:
-                            start_sigma[icrsh][spin_channel] << (1+fac)*dc_value
+                            start_sigma[icrsh][spin_channel] << -fac
                         else:
-                            start_sigma[icrsh][spin_channel] << (1-fac)*dc_value
+                            start_sigma[icrsh][spin_channel] << fac
```

Comment:
I think this breaks the initialization of Sigma to dc_value

Reply:
Added temporary fix: +dc_value